index.html

```diff
@@ -80,3 +89,221 @@ <h2 class="secondary-header">LET’S CONNECT</h2>
     </section>
   </body>
 </html>
+<!-- WORKS SECTION  -->
+<section class="works">
+  <!-- first project  -->
+ <div class="projects container">
+    <img class="project-img" src="images/tonic.jpeg" alt="" />
+    <h3 class="tertiary-heading">Tonic</h3>
+    <ul class="project-details">
+      <li>CONOPY</li>
+      <li>Back End Dev</li>
+      <li>2015</li>
+    </ul>
+    <p class="project-description">
```

Comment:
- [ ] Please check my feedback throughout the video I recorded for you [here](https://youtu.be/dfA1Wjs-Xi4) 😉

Reply:
I have make the changes